### PR TITLE
[5.4.x] Change file creation method from copy to create #3283

### PR DESCRIPTION
**Pull request by btyanagawamg**
Please review #3283 .

This PR is backport for 5.4.x .

### CODE REVIEW

File: source/Introduction/ChangeLog.rst [hunk 1]
```diff
@@ -59,6 +59,12 @@
 
         * \ ``SPRING_SECURITY_LAST_EXCEPTION`` \ が格納されるスコープの誤記を修正
 
+    * -
+      - :doc:`../Tutorial/TutorialREST`
+      - 記載内容の修正(管理ID#3283 )
+
+        * spring-mvc-rest.xmlを作成する方法の説明を変更
```
Comment (yoshikawaa) [inline source/Introduction/ChangeLog.rst]: - [ ] 5.4.1 -> 5.4.2

Comment (btyanagawamg) [reply source/Introduction/ChangeLog.rst]: beb7931b18a2e4cbef6cd4c8b9a111e2a32af305 
Fixed at above commit .

**Comment by yoshikawaa** [comment]
@btyanagawamg Please resolve conflict.

**Comment by btyanagawamg** [comment]
I fixed conflict .

**Pull request merged**